rhacmstackem.sh

```diff
@@ -80,27 +80,27 @@ if [[ "${RBAC_SETUP:-"true"}" == "true" ]]; then
   echo "$(date) ##### Setting up RBAC users"
   export RBAC_PASS=$(head /dev/urandom | tr -dc 'A-Za-z0-9' | head -c $((32 + RANDOM % 8)))
   export KUBECONFIG=${LIFEGUARD_PATH}/clusterclaims/${CLUSTERCLAIM_NAME}/kubeconfig
-  touch ./rbac/htpasswd
+  touch ./resources/rbac/htpasswd
```

Comment:
Not that it's a ton of places, but would you mind putting this in a variable? Past me thanks you 🙂 
```suggestion
  RBAC_DIR="./resources/rbac"
  HTPASSWD_FILE="${RBAC_DIR}/htpasswd'
  touch ${HTPASSWD_FILE}
```

Reply:
Good idea!